cmake/openenclave-config.cmake.in

```diff
@@ -94,7 +94,7 @@ if (NOT TARGET openenclave::sgx_dcap_ql)
   find_library(SGX_DCAP_QL_LIB NAMES sgx_dcap_ql
                HINTS ${NUGET_PACKAGE_PATH}/DCAP_Components/build/lib/native/Libraries)
   if (NOT SGX_DCAP_QL_LIB)
-    message(FATAL_ERROR "-- Looking for sgx_dcap_ql library - not found")
+    message(WARNING "-- Looking for sgx_dcap_ql library - not found")
```

Comment:
Could we expand this to say attestation tests and samples will fail without this?

Reply:
Fixed